printer_mods/README.md

```diff
@@ -12,6 +12,7 @@ Discord to have your mod moved to this folder.
 ## Table structure
 
 Update the following table with the information about your mod:
+
```

Comment:
Looks like there are several unrelated modifications to this README, please limit changes here to *only* your entry